aten/src/ATen/native/cpu/ScatterGatherKernel.cpp

```diff
@@ -333,6 +527,8 @@ void scatter_scalar_reduce_cpu_kernel(const Tensor& self, const int64_t dim, con
     cpu_scatter_gather_base_kernel<>()(self, dim, index, value,
                                        "scatter_scalar_reduce_multiply_", reduce_multiply);
     break;
+  default:
```

Comment:
Why did you add this?

Reply:
Without this the build on CI will fail. I think it's due to this 

> Using the -Werror compiler flag, a switch statement over a value of an enum type without a default label will fail to compile if any enumerator of the enum doesn’t have a corresponding case. This is sometimes called an exhaustive or defaultless switch statement.